### DISCUSSION TITLE
How to use Emgu.CV.Stitching.MultiBandBlender ???

### DISCUSSION THREAD

**Discussion by eNovware** [emgucv/emgucv#707] [Q&A] [answered]
Hi world,

I'm a rookie emgu user and I want to use the _**Emgu.CV.Stitching.MultiBandBlender**_ class for merging multiple images.

I have found the dreamed tool into openCV : _**cv::detail::MultiBandBlender**_
( https://docs.opencv.org/4.5.5/d5/d4b/classcv_1_1detail_1_1MultiBandBlender.html )

But the emgu class don't have any method where opencv ??!
( https://www.emgu.com/wiki/files/4.1.0/document/html/029920b5-1987-5338-4fa5-7493a25cc58d.htm )

Does anybody have an idea about how to use this ??

Thanks a lots.
Seb.

**Comment by emgucv** [comment, marked as answer]
Added Prepare, Feed & Blend function to Blender in this commit:
2367f6497ac793261ea56be7c96d41ea7583aeab

It will be included in the next release.

**Reply by eNovware** [reply]
Many thanks for your fast action !
Amazing !!!!!!
